scene/resources/3d/primitive_meshes.cpp

```diff
@@ -421,13 +421,21 @@ void CapsuleMesh::create_mesh_array(Array &p_arr, const float radius, const floa
 	float height_v = (height - 2.0 * radius) / vertical_length; // v size of height section
 
 	// note, this has been aligned with our collision shape but I've left the descriptions as top/middle/bottom
-
-	Vector<Vector3> points;
-	Vector<Vector3> normals;
-	Vector<float> tangents;
-	Vector<Vector2> uvs;
-	Vector<Vector2> uv2s;
-	Vector<int> indices;
+	int num_points = (rings + 2) * (radial_segments + 1) * 2;
+	LocalVector<Vector3> points;
+	points.reserve(num_points);
```

Comment:
These `Vector` instances all seem to be with trivial types; how about continuing to use `Vector` but using `resize` and `ptrw()` assigning instead? That would save you from having to copy over elements at the end :)
(we really need `reserve` for `Vector`...)

Reply:
See above discussion

Reply:
Oh, whoops, skipped over too much there 😄 I guess then it's fine!
How about adding comments that quickly explain the type oddity?

Reply:
Explaining why we use LocalVector then copy over to Vector at the end you mean?

Reply:
Alright, added a comment!

Reply:
Perfect, thanks!